src/utils.ts

```diff
@@ -60,6 +60,11 @@ export function getOAuthSettings(): Promise<ClaspToken> {
     });
 }
 
+// Default settings for clasp.json
+export const DEFAULT_SETTINGS = {
+  SRC_PATTERNS: ['src/**','appsscript.json'],
```

Comment:
Source code is not necessarily in 'src/'.  (Ex: dist / Code.js)
It is specified by `rootDir` of `.clasp.json` .

See also : https://github.com/google/clasp#rootdir-optional